homeassistant/components/zwave_js/sensor.py

```diff
@@ -51,6 +51,7 @@
 from homeassistant.helpers.entity_platform import AddEntitiesCallback
 from homeassistant.helpers.typing import UNDEFINED, StateType
 
+from .binary_sensor import is_valid_binary_notification_sensor
```

Comment:
This isn't great since we don't want platforms to depend on each other. The approach in Matter would solve this but that's a large refactor not for now.

I don't have a better suggestion at the moment so let's keep it for now.

Reply:
Yeah, I was a bit in doubt about this one - we could move it into a util module
In the end the check is super simple but still risky to duplicate